config/initializers/rswag_ui.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Rswag::Ui.configure do |c|
+  # List the Swagger endpoints that you want to be documented through the
+  # swagger-ui. The first parameter is the path (absolute or relative to the UI
+  # host) to the corresponding endpoint and the second is a title that will be
+  # displayed in the document selector.
+  # NOTE: If you're using rspec-api to expose Swagger files
+  # (under swagger_root) as JSON or YAML endpoints, then the list below should
+  # correspond to the relative paths for those endpoints.
+
+  c.swagger_endpoint "/api-docs/openapi#{ENV.fetch('TEST_ENV_NUMBER', '')}.yaml", 'API V1 Docs'
+
+  # Add Basic Auth in case your API is private
+  c.basic_auth_enabled = true
```

Comment:
What if we have both? public and private endpoints?

Reply:
Is the way that `rswag` works, we need to define the API endpoints that will use it for UI section

Reply:
Not sure if that answers my question.
If we build an app that has public endpoints, we need to also have a "public" documentation that anyone can see in order to understand how the API works.

So the question is, is it possible to only scope the auth to a subset of our endpoints and not the whole app?

Reply:
@JulianPasquale Oh Oh, I understood that you are asking about `rswag` sorry

- I'm not sure, but I looked in the Swagger documentation and didn't find anything about adding auth only for a group of endpoints, also following the ticket scope I included the authorization here, but we can disable it by setting the value to "false". `

Reply:
Cool. I think it would be nice to explore it so we can allow the mix of public and private docs. Not a blocker since auth can be disabled 👍

Reply:
I think this is a good discussion, maybe we can follow up in a meeting? I think what we need to answer is: who are we expecting to be the consumers of these endpoints?
- Frontend devs?
- Backend devs from other teams/apps?
- External products/services?